org.thingpedia.iot.connectivity/manifest.tt

```diff
@@ -0,0 +1,13 @@
+abstract class @org.thingpedia.iot.connectivity
+#_[thingpedia_name="Connectivity Sensor"]
+#_[thingpedia_description="Interface for Connectivity Sensor."]
+#_[description="Interface for Connectivity Sensor."]
+#[license="BSD-3-Clause"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(connected,disconnected))
+  #_[canonical="connectivity sensor state"]
+  #_[confirmation="the state of $__device"]
+  #_[formatted=["It is $state."]];
+}
```

Comment:
Have we figured out how this sensor is supposed to be used? "Connectivity sensor" is not a commonly used expression in English...

Reply:
Hmmm not really. I also asked about some of the other weirder sensors but Paulus doesn't seem to be sure either. Should we remove this?

Reply:
Yeah let's keep it on hold until we know what it is.

Reply:
Dropped in latest commit